lepton/package.go

```diff
@@ -32,11 +32,11 @@ type PackageList struct {
 
 // Package is the definition of an OPS package.
 type Package struct {
-	Runtime     string `json:"runtime"`
-	Version     string `json:"version"`
-	Language    string `json:"language"`
+	Runtime     string `json:"runtime,omitempty"`
+	Version     string `json:"version,omitempty"`
+	Language    string `json:"language,omitempty"`
 	Description string `json:"description,omitempty"`
-	SHA256      string `json:"sha256"`
+	SHA256      string `json:"sha256,omitempty"`
```

Comment:
As far as I understand, the `omitempty` directive is only used when marshaling data into JSON strings, and not when unmarshaling JSON strings, so I don't think it makes a difference here.

Reply:
you're right - it was actually just the invalid json that was blowing things up - rmd